src/components/header/header.tsx

```diff
@@ -43,6 +44,37 @@ const StyledHeader = styled.header<{
   }
 `;
 
+const LanguageSwitch = styled.button<{ selected?: boolean }>`
```

Comment:
When do we need a `LanguageSwitch` and when a `StyledLanguageLink`? 

I checked the header on the HomePage, and there the `LanguageSwitch` is implemented. Those are buttons. But as they are links, it should be an `a` Tag. Otherwise it's not accessible*.

*I'm not 100% sure how to implement language switches perfectly from an a11y perspective. Let's check some popular pages how they have implemented it.